lib/applicants/resolvers.ts

```diff
@@ -9,7 +9,14 @@ import { formatPhoneNumber, formatPostalCode } from '@lib/utils/format';
  * @returns All RCD applicants
  */
 export const applicants: Resolver = async (_parent, _args, { prisma }) => {
-  const applicants = await prisma.applicant.findMany();
+  const applicants = await prisma.applicant.findMany({
+    include: {
+      applications: true,
+      permits: true,
+      guardian: true,
+      medicalInformation: true,
+    },
+  });
```

Comment:
We don't want to include all of these fields off the bat. The goal of GraphQL is to only do as much work as needed, so we don't want to do all these joins and queries if we're not asking for these fields.

Instead, we want to make use of field resolvers to retrieve these nested fields, like so (from Ben Awad video lol):
![image](https://user-images.githubusercontent.com/30249230/125838843-28a769ff-9c18-4ea4-b9f1-c587711eb3a6.png)

We'll address N+1 problem later.